### PR TITLE
Add the CRD of the application

### PR DESCRIPTION
**Pull request by LinuxSuRen**
As mentioned in the title.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 lint-chart:
 	helm lint charts/ks-devops
 
+remove-crds:
+	kubectl delete -f charts/ks-devops/crds
```
Comment (JohnNiang) [inline Makefile:5]: How about add this command line into here?

https://github.com/kubesphere-sigs/ks-devops-helm-chart/blob/5e8f988105f1360bf2d196d1e5a1a1739f19353f/Makefile#L8-L10

Comment (JohnNiang) [reply Makefile:5]: But this won't prevent this PR from merging.

Comment (LinuxSuRen) [reply Makefile:5]: I'm not sure about that. For the Helm chart, the default behavior is not deleting the CRDs. I prefer not to break the default behavior. They can explicitly use the command `make remove-crds`.

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Pull request merged**